Terminal.Gui/Views/DateField.cs

```diff
@@ -47,9 +48,11 @@ public class DateField : TextField {
 		/// <param name="y">The y coordinate.</param>
 		/// <param name="date">Initial date contents.</param>
 		/// <param name="isShort">If true, shows only two digits for the year.</param>
-		public DateField (int x, int y, DateTime date, bool isShort = false) : base (x, y, isShort ? 10 : 12, "")
+		/// <param name="isJalaali">If true, parse will convert jalaali input fo georgian date</param>
+		public DateField (int x, int y, DateTime date, bool isShort = false, bool isJalaali = false) : base (x, y, isShort ? 10 : 12, "")
```

Comment:
I propose that you move `bool isJalaali` out of `DateField` into `Application` and make it an app wide setting.

That way we avoid cluttering constructors with more parameters and we allow for additional controls that need Jalaali support.

What do you think of that?

Reply:
it's good idea, you mean I move `isJalaali' as a static field  into `Application.cs` and use it inside the `DateField` class like this: 
```
if(Application.IsJallali == true){
....
}
```
is that  right?

Reply:
Yes.

Reply:
> Yes.

what if we need to use both Georgian and Jalaali inputs date in an application at the same time? in some situations we have a form that needs to enter multiple date inputs in different formats.

Reply:
Good point.